exercises.py

```diff
@@ -1,13 +1,168 @@
+#  The variables are destroyed.
+# 7. What is a return value? Can a return value be part of an expression?
+#  The return value is the value that a function calls. Yes, a return value can be used in an expression.
+# 8. If a function does not have a return statement, what is the return value of a call to that function?
+#  None
+# 9. How can you force a variable in a function to refer to the global variable?
+#  Use a global statement in the function.
+# 10. What is the data type of None? 
+#  NoneType
+# 11. What does the import areallyourpetsnamederic statement do? 
+#  It will import the module areallypourpetsnamederic to python so that phython will be able to understand it when ran.
+# 12. If you had a function named bacon() in a module named spam, how would you call it after importing spam? 
+#  spam.bacon()
+# 13. How can you prevent a program from crashing when it gets an error? 
+#  You would use the try and except statement. 
+# 14. What goes in the try clause? What goes in the except clause? 
+#  The valid arguements go in the try clause and the errored argument goes within the except statement.
+
+# # Chapter 4
+# 1. What is []?
+#  Denotes a list, and item stored inside theses bracket are list items.
+# 2. How would you assign the value 'hello' as the third value in a list stored in a variable named spam? (Assume spam contains [2, 4, 6, 8, 10].)
+#  spam = [2,4,'hello', 6, 8]
+# For the following three questions, let’s say spam contains the list ['a', 'b', 'c', 'd'].
+# 3. What does spam[int(int('3' * 2) // 11)] evaluate to? 
+# 'd'
+# 4. What does spam[-1] evaluate to? 
+# 'd'
+# 5. What does spam[:2] evaluate to?
+#  ['a', 'b', 'c']
+# For the following three questions, let’s say bacon contains the list [3.14, 'cat', 11, 'cat', True].
+# 6. What does bacon.index('cat') evaluate to?
+#  1
+# 7. What does bacon.append(99) make the list value in bacon look like? 
+#  bacon=[3.14, 'cat', 11, 'cat', True, 99]
+# 8. What does bacon.remove('cat') make the list value in bacon look like? 
+#  bacon=[3.14, 11, 'cat', True]
+# 9. What are the operators for list concatenation and list replication? 
+#  The operators are + to combine values and * to replicate list.
+
+# 10. What is the difference between the append() and insert() list methods? 
+#  append() will add an item to the end of a list and insert() allows one to choose where exactly in a list an item will go.
+
+# 11. What are two ways to remove values from a list? 
+#  remove() and del()
+
+# 12. Name a few ways that list values are similar to string values. 
+#  Both are called items, can be store as variables, and have to be contained within parameteres []for list and ''or ""for a string value.
+
+# 13. What is the difference between lists and tuples? 
+#  List use [], while tiples use (). Tuplues can not be modified, appended or removed.
+# 14. How do you type the tuple value that has just the integer value 42 in it?
```

Comment:
42 is an integer by default, so you don't need to adjust the type. What does the tuple need besides the parentheses in order for Python to know it is a tuple?